modules/smaatoBidAdapter.js

```diff
@@ -196,6 +197,22 @@ export const spec = {
    * @return {UserSync[]} The user syncs which should be dropped.
    */
   getUserSyncs: (syncOptions, serverResponses, gdprConsent, uspConsent) => {
+    if (syncOptions && syncOptions.pixelEnabled) {
```

Comment:
you forgot about gpp? Also, why not import https://github.com/prebid/Prebid.js/blob/8f817bd187de3a1bde065f85eb5b6e0a6b034b66/libraries/teqblazeUtils/bidderUtils.js#L206 ?

Reply:
the changes are sufficient for now, but thank you for checking.

regarding the generic implementation: both options currently don't fit our use case. if those should be used, i can of course rework one of them to make it fit. is there some preference which one should be worked on for a generic implementation?

Reply:
No preference, I'll merge this and keep we'll study some more adapters before starting a generic implementation